pkg/providers/amifamily/resolver.go

```diff
@@ -64,12 +64,18 @@ type Options struct {
 // LaunchTemplate holds the dynamically generated launch template parameters
 type LaunchTemplate struct {
 	*Options
-	UserData            bootstrap.Bootstrapper
-	BlockDeviceMappings []*v1alpha1.BlockDeviceMapping
-	MetadataOptions     *v1alpha1.MetadataOptions
-	AMIID               string
-	InstanceTypes       []*cloudprovider.InstanceType `hash:"ignore"`
-	DetailedMonitoring  bool
+	UserData              bootstrap.Bootstrapper
+	BlockDeviceMappings   []*v1alpha1.BlockDeviceMapping
+	MetadataOptions       *v1alpha1.MetadataOptions
+	AMIID                 string
+	InstanceTypes         []*cloudprovider.InstanceType `hash:"ignore"`
+	DetailedMonitoring    bool
+	LicenseSpecifications []string
+	Placement             Placement
```

Comment:
should be able to use `v1alpha1.Placement`

```suggestion
	Placement             *v1alpha1.Placement
```